src/main/java/command/DeadlineCommand.java

```diff
@@ -0,0 +1,25 @@
+package command;
+
+import java.time.LocalDateTime;
+
+import henry.Task;
+
+/**
+ * Responsible for adding Deadline tasks to the task list.
+ * Deadline tasks are tasks with a deadline date and time.
+ */
+public class DeadlineCommand extends Command {
+
+    public static final String COMMAND_WORD = "deadline";
+    public static final String MESSAGE_SUCCESS = "OK. I ADDED THIS TASK TO MY LIST:\n\t\t\t %1$s.";
```

Comment:
perhaps make this static variable private if you only intend on using it for this class?

Reply:
Good spot, the success messages have all been made private!